loadingScreen.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  loadingScreen.swift
+//  nameToFaces
+//
+//  Created by Felipe Gil on 2021-06-28.
+//
+
+import UIKit
+
+class LoadingViewController: UIViewController {
+    var loadingActivityIndicator: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView()
+            indicator.style = .large
+            indicator.color = .white
+            indicator.startAnimating()
+            indicator.autoresizingMask = [.flexibleLeftMargin, .flexibleRightMargin, .flexibleTopMargin, .flexibleBottomMargin]
+            return indicator
+    }()
+    
+    var blurEffectView: UIVisualEffectView = {
+        let blurEffect = UIBlurEffect(style: .dark)
+            let blurEffectView = UIVisualEffectView(effect: blurEffect)
+            blurEffectView.alpha = 0.8
+            blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+            return blurEffectView
+        }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.black.withAlphaComponent(0.5)
+        blurEffectView.frame = self.view.bounds
+        view.insertSubview(blurEffectView, at: 0)
+        loadingActivityIndicator.center = CGPoint(x: view.bounds.midX, y: view.bounds.midY)
+        view.addSubview(loadingActivityIndicator)
```

Comment:
When using frames (or center) to determine size and position of views, you're dropping auto layout - and all benefits from it (like automatic update when rotating the device). Instead you should use constraints for that. To achieve the same you did here, you'd have this:

```suggestion
        blurEffectView.translatesAutoresizingMaskIntoConstraints = false // This is an explicit declaration that auto resizing masks won't be used
        loadingActivityIndicator.translatesAutoresizingMaskIntoConstraints = false
        view.addSubview(blurEffectView)
        view.addSubview(loadingActivityIndicator) 

        NSLayoutConstraints.activate([
            // the lines below basically say that all edges of the blur effect view will be pinned to the view's ones
            blurEffectView.topAnchor.constraint(equalTo: view.topAnchor),
            blurEffectView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
            blurEffectView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
            blurEffectView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
            
            // and these place the indicator centered in the view at all times
            loadingActivityIndicator.centerYAnchor.constraint(equalTo: view.centerYAnchor),
            loadingActivityIndicator.centerXAnchor.constraint(equalTo: view.centerXAnchor)
        ])                
```

Here is a good reference about it: https://www.avanderlee.com/swift/auto-layout-programmatically/